### PR TITLE
Ability to add/remove Owners from a Story

### PR DESCRIPTION
**Pull request by mattvv**
Starting to make a pseudo state element for an edited story so we can edit different elements. @RamiroPinol what do you think of this direction?

### CODE REVIEW

File: components/StoryModal.tsx [hunk 1]
```diff
@@ -29,26 +32,36 @@ const Section = ({ title, children }): JSX.Element => (
...
     });
     dispatch(editStory(newStory));
-    setDescription('');
+    setEditedFields(getEditableFields(newStory));
   });
 
   const handleClose = () => {
-    if (description) {
-      saveDescription();
+    if (EDITABLE_FIELDS.some(field => editedFields[field] !== story[field])) {
```
Comment (mattvv) [inline components/StoryModal.tsx:63]: If any 1 of the fields is changed we should save

**Comment by RamiroPinol** [comment]
@mattvv I like the approach! It will make the save functionality more consistent.

**Comment by mattvv** [comment]
Awesome! I'll continue work on this PR then to add owners

**Comment by mattvv** [comment]
Ready for review!

File: components/StoryModal.tsx [hunk 2]
```diff
@@ -29,43 +32,71 @@ const Section = ({ title, children }): JSX.Element => (
...
+
+  useEffect(() => {
+    setEditedFields(getEditableFields(story));
+  }, [story?.id]);
 
-  const [_, saveDescription] = usePivotal(async ({ apiKey, projectId }) => {
+  const [_, saveStory] = usePivotal(async ({ apiKey, projectId }) => {
+    const payload = {
+      description: editedFields.description,
+      owner_ids: editedFields.owners.map(owner => owner.id),
```
Comment (mattvv) [inline components/StoryModal.tsx:58]: gross but pivotal doesn't accept the owner objects

**Review by evanpettie — APPROVED** [review]

**Review by jansen0825 — APPROVED** [review]

**Review by RamiroPinol — APPROVED** [review]

**Pull request merged**